pom.xml

```
@@ -34,7 +34,7 @@
   </parent>
 
   <artifactId>bouncycastle-api</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>${revision}-${changelist}</version>
```

as is this would create versions lower than we have today.
in order to keep the cd versions newer we would need to keep a prefix like the following: 
```suggestion
  <version>2.30.${revision}-${changelist}</version>
```

if  when the bouncycastle API produces a version > 2.30 we would then be able to remove the prefix and directly use the bouncycastle version.